brightchain-lib/src/lib/documents/base/document.ts

```
@@ -0,0 +1,129 @@
+  /**
+   * Clear private data provider
+   */
+  public clearPrivateDataProvider(): void {
+    this.privateDataProvider = undefined;
+  }
+
+  /**
+   * Save document to storage
+   * To be implemented by subclasses
+   */
+  public async save(operational: TOperational): Promise<void> {
```

ESLint

[error] 'operational' is defined but never used.

(@typescript-eslint/no-unused-vars)